doc/sphinx/source/developer_guide/guidance_stabilization.rst

```diff
@@ -0,0 +1,57 @@
+.. developer_guide guidance_stabilization
+
+===========================
+Guidance and Stabilization
```

Comment:
This page is not finished, maybe better to keep it out for now. Also, I think it should be split into a `user` part and a `dev` part:
- general structure and modes for user
- implementation of control for dev
- also split stabilization and guidance ?
- same pages for rotorcraft and fixedwing (and rover...) ?

Reply:
this one came from the branch matteo's is working on still. I will remove it. 

Regarding rotorcraft and fixedwing, I am working on the airframe section in the user guide, in PR soon